src/base/baseAnnotationTool.js

```diff
@@ -15,25 +15,37 @@ export default class extends baseTool {
       configuration,
       supportedInteractionTypes
     });
-
-    this.isAnnotationTool = true;
```

Comment:
Good call on removing this 👍 
It's one of those things I put in "just because", but never ended up using anywhere.